### ISSUE TITLE
The semantic hint should be generated from overriden actions on iOS if no default hint provided.

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#19701]

**Comment by zoechi** [comment]
The PR #5823 referencing this issue is merged. 
Is there still something to do?

**Comment by pedromassangocode** [comment]
Hi @jonahwilliams 
Can you confirm that this still a valid issue?
Thank you

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
